packages/skeleton/src/lib/components/Radio/RadioGroup.svelte

```diff
@@ -47,8 +47,6 @@
 	const cBase = 'p-1';
 
 	// Reactive
-	// display space-x only on row flex as it is not needed on column flex.
-	$: spacing = `${display.includes('flex-col') ? '' : 'space-x-1'}`;
```

Comment:
@phamduylong we still need this because of this bug: #1672
I would suggest the following:
```ts
export let spacing: CssClasses = 'space-x-1';
//....
// display space-x only on row flex as it is not needed on column flex.
$: classesSpacing = `${display.includes('flex-col') ? '' : spacing}`;
$: classesBase = `${cBase} ${display} ${background} ${border} ${classesSpacing} ${rounded} ${$$props.class ?? ''}`;
```